IPPDotNetDevKitCSV3/Code/Intuit.Ipp.Core/Configuration/JsonFileConfigurationProvider.cs

```diff
@@ -445,6 +445,14 @@ public IppConfiguration GetConfiguration()
 
             if (!string.IsNullOrEmpty(messageRequestSettings["CompressionFormat"]))
             {
+                if (messageRequestSettings["CompressionFormat"].ToLower() == CompressionFormat.GZip.ToString().ToLower())
```

Comment:
Rather than compare ToLower strings, please use the StringComparison overload
`if (messageRequestSettings["CompressionFormat"].Equals(CompressionFormat.GZip.ToString(), StringComparison.OrdinalIgnoreCase)`

Reply:
Thanks for the feedback. We probably might not merge this pull request to master anytime soon as this is on hold. But definitely, thanks for the feedback.